lib/hammer_cli_katello/capsule.rb

```diff
@@ -22,39 +22,9 @@ def all_options
       end
     end
 
-    class ListCommand < HammerCLIKatello::ListCommand
-      action :index
-
-      output do
-        field :id, _("Id")
-        field :name, _("Name")
-        field :url, _("URL")
-      end
-
-      build_options
-    end
-
-    class InfoCommand < HammerCLIKatello::InfoCommand
-
-      action :show
-
-      output ListCommand.output_definition do
-        field :_features, _("Features"),   Fields::List
-        field :created_at, _("Created at"), Fields::Date
-        field :updated_at, _("Updated at"), Fields::Date
-      end
-
-      def extend_data(proxy)
-        proxy['_features'] = proxy['features'].map { |f| f['name'] }
-        proxy
-      end
-
-      build_options
-    end
-
-    class Content < HammerCLI::AbstractCommand
+    class Content < HammerCLIKatello::Command
       command_name 'content'
-      desc _('Manage the capsule content')
```

Comment:
I'm in favor of leaving the description set here, so that ``#subcommand()`` down below can reference it. One place to change the description in the future, rather than two.